app/models/product_file.rb

```diff
@@ -48,6 +50,7 @@ class ProductFile < ApplicationRecord
   after_commit :schedule_rename_in_storage, on: :update, if: :saved_change_to_display_name?
 
   attr_json_data_accessor :epub_section_info
+  attr_json_data_accessor :isbn
```

Comment:
Let's avoid using JSON accessors for simple values. JSON accessors should be used for varying JSON strutures that we don't need to query.
Please use a database column instead.